### DISCUSSION TITLE
windows系统用VS2017或VS2012生成的应用搜索速度只有你的25%左右

### DISCUSSION THREAD

**Discussion by topontop** [community/community#33141] [Programming Help]
能编译、链接成功。
AVX2版本，32核心CPU，编译参数逐个测试，NPS从最初的4300K左右提高到6300K就上不去了。求指导！
和你生成的应用比较，一个是文件大小不一样，比你的小不少，重要是你的NPS大约有26000k左右。
附上编译优化参数：
/MP /ifcOutput "x64\Release\" /GS /TP /GL /W2 /Gy /Zc:wchar_t /Zi /Gm- /O2 /Ob1 /sdl /Fd"x64\Release\vc143.pdb" /Zc:inline /fp:fast /D "_UNICODE" /D "UNICODE" /errorReport:prompt /GF /GT /WX- /Zc:forScope /std:c17 /arch:AVX2 /Gr /Oy /Oi /MT /std:c++17 /FC /Fa"x64\Release\" /EHsc /nologo /Fo"x64\Release\" /Ot /Fp"x64\Release\Pikafish.pch" /diagnostics:column 
-DNDEBUG -DNO_PREFETCH -DUSE_POPCNT -DUSE_PEXT

**Comment by mster1222** [comment]
git@github.com:mster1222/protocol.git

**Comment by otter-computer** [comment]
Hi @topontop, thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
